volume-providers/softlayer/utils/storage_utils.go

```diff
@@ -31,6 +31,10 @@ var ENDURANCE_TIERS = map[string]int{
 	"10":   1000,
 }
 
+const (
+	SoftLayerFileTypeName = "file"
```

Comment:
I think we should not declare any specific volume type here as this file is applicable for both file and block. May be we can have separate file for such constants

Reply:
Done